ZeeHomeScreen/Classes/Factory/ComponenttFactory.m

```diff
@@ -178,6 +131,11 @@ + (NSString *)viewControllerClassNameforType:(NSString *)type {
             }
         }
     }
+
+    if (viewController == nil) {
```

Comment:
moved it here due to removing method above

Reply:
SRP fix.